### PR TITLE
ReleaseCandidate 1

### PR DESCRIPTION
**Pull request by nlzh**
wrapped up procedure-PR;
added info-gu, extended environment.

### CODE REVIEW

**Comment by nlzh** [comment]
Last changes are bugfixes, found on 4.4

File: classes/class.ilObjJill.php [hunk 1]
```diff
@@ -42,7 +42,33 @@ function doRead() {
...
+			$local_settings = array();
+			$local_fieldnames = $package_settings->getInputNames();
+
+			foreach ($local_fieldnames as $local_fieldname) {
+				$type = $package_settings->getTypeOf($local_fieldname);
+				$local_field = new Jill\Settings\ObjectPackageDefinitions\Field(
+					//$type, $field, $value
+					$type, $local_fieldname, null
+				);
+				//array_push($local_settings, $local_field);
```
Comment (klees) [inline classes/class.ilObjJill.php]: No uncommented code please.

File: classes/ilActions.php [hunk 2]
```diff
@@ -88,11 +88,22 @@ public function savePackageSettings($post) {
 			$settings[] = new Settings\ObjectPackageDefinitions\Field($type, $name, $value);
 		}
 
+		$this->field_settings_db->store(
```
Comment (klees) [inline classes/ilActions.php]: `store` ain't defined on `$this->field_settings_db`.

**Comment by klees** [comment]
Tests still are completely missing.

**Pull request merged**